packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1605,28 +1606,30 @@ describe('network stubbing', { retries: 2 }, function () {
         .wait('@err', { timeout: 50 })
       })
 
-      it('can timeout in req.reply handler', {
-        defaultCommandTimeout: 50,
-      }, function (done) {
+      it('can timeout in req.reply handler', function (done) {
         cy.on('fail', (err) => {
+          Cypress.config('defaultCommandTimeout', 5000)
           expect(err.message).to.match(/^A response callback passed to `req.reply\(\)` timed out after returning a Promise that took more than the `defaultCommandTimeout` of `50ms` to resolve\./)
```

Comment:
sometimes this can take more than 50ms due to command log lag :upside_down_face: